docs/customize.md

```diff
@@ -167,7 +167,7 @@ Disables gutter icons when doing multiple selections.
 ```
 
 ## Tag Plugin Settings
-Tag settings found in `bh_core.sublime-settings`.
+Tag settings found in `bh_tag.sublime-settings`.
```

Comment:
I recently pushed them into their own file as they are actual settings for the bh_tag plugin and not the core.   Just forgot to update the docs.